website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +86,60 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($('table.responsive-table-xxs thead').is(':hidden')) {
+        $('table.responsive-table-xxs tbody tr td:first-child.expanded').siblings().children().css('display', 'block');
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').off().on('click', function() {
```

Comment:
Depending on these CSS selectors is brittle; can you use knockout for this instead?